executor/adapter.go

```diff
@@ -333,6 +333,12 @@ func (a *ExecStmt) buildExecutor(ctx sessionctx.Context) (Executor, error) {
 var QueryReplacer = strings.NewReplacer("\r", " ", "\n", " ", "\t", " ")
 
 func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
+	sessVars := a.Ctx.GetSessionVars()
+	if !sessVars.InRestrictedSQL {
+		for access := range sessVars.StmtCtx.AccessDigest {
+			metrics.IndexAccessCounter.WithLabelValues(access.Tbl, access.Idx).Inc()
```

Comment:
So we are counting the table/index access in query level,  what if we are executing a complicated query on a big table? the query count is 1 but we may spend a long time on this table/index, or we may access the table/index more than once in this single query. My question is how to understand the `hot` table/index here.